src/System.Management.Automation/engine/remoting/common/WireDataFormat/RemoteSessionCapability.cs

```diff
@@ -121,19 +112,15 @@ internal static byte[] GetCurrentTimeZoneInByteFormat()
                 {
                     s_timeZoneInByteFormat = Utils.EmptyArray<byte>();
                 }
-#endif
             }
 
             return s_timeZoneInByteFormat;
         }
 
-#if !CORECLR // TimeZone Not In CoreCLR
         /// <summary>
         /// Gets the TimeZone of the destination machine. This may be null
         /// </summary>
         internal TimeZone TimeZone { get; set; }
```

Comment:
I see that BinaryFormatter type is now available in Core, good news!  Since this property is now available in Core can you also remove the "#if !CORECLR" at the two places it is referenced (serverremotesession.cs and EncodeAndDecode.cs)?  
https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/remoting/common/WireDataFormat/EncodeAndDecode.cs#L2405
https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/remoting/server/serverremotesession.cs#L747